app/models/user.rb

```diff
@@ -1,5 +1,6 @@
 class User < ActiveRecord::Base
   has_many :rides
+  has_many :passengers
```

Comment:
I will be starting a new branch for the change of the relationship model names. Cause it will probably require some new tests.
See: https://github.com/websitewizardguy/travelrogue/tree/feature/change_relation_names